pygfx/geometries/_base.py

```diff
@@ -142,7 +142,7 @@ def get_bounding_box(self):
             if self._aabb_rev == self.grid.rev:
                 return self._aabb
             # account for multi-channel image data
-            grid_shape = self.grid.data.shape[: self.grid.dim]
+            grid_shape = tuple(reversed(self.grid.size[: self.grid.dim]))
```

Comment:
This fixes the bounding box logic so it also works with textures that don't own data.